vipul/Gallery_System/bird.html

```diff
@@ -0,0 +1,105 @@
+						<ul class="dropdown">
+							<li><a href="wild.html">Wildlife</a></li>
+							<li><a href="landscape.html">Landscape</a></li>
+							<li><a href="portrait.html">Portrait</a></li>
+							<li><a href="fashion.html">Fashion</a></li>
+							<li><a href="macro.html">Macro</a></li>
+							<li><a href="bird.html">Bird</a></li>
+							<li><a href="nature.html">Nature</a></li>
+							<li><a href="food.html">Food</a></li>
+						</ul>
+					</li>
+					<li><a href="services.html">SERVICES</a></li>
+					<li><a href="about.html">ABOUT</a></li>
+					<li><a href="contact.html">CONTACT</a></li>
+				</ul>
+			</div>
+		</div>
+		<div class="category-wild">
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird.jpg">
+						<img src="images/bird.jpg" alt="Picture of Bird Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird1.png">
+						<img src="images/bird1.png" alt="Picture of Bird Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird2.jpg">
+						<img src="images/bird2.jpg" alt="Picture of Bird Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird3.jpg">
+						<img src="images/bird3.jpg" alt="Picture of Bird Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird4.jpg">
+						<img src="images/bird4.jpg" alt="Picture of Bird Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird5.jpg">
+						<img src="images/bird5.jpg" alt="Picture of Bird Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird6.jpg">
+						<img src="images/bird6.jpg" alt="Picture of Bird Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird7.png">
+						<img src="images/bird7.png" alt="Picture of Bird Photography">
+					</a>
+			</div><div class="gallery-wild">
+					<a target="_blank" href="images/bird8.jpg">
+						<img src="images/bird8.jpg" alt="Picture of Bird Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird9.jpg">
+						<img src="images/bird9.jpg" alt="Picture of Bird Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird.jpg">
+						<img src="images/bird.jpg" alt="Picture of Bird Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/bird1.png">
+						<img src="images/bird1.png" alt="Picture of Bird Photography">
+					</a>
+			</div>
+		</div>
+		<div class="footer">
+			<p>
+				Copyright 2020 All rights reserved | This is developed by <a href="https://www.facebook.com/vipul.suryawanshi" target="_blank">Vipul Suryawanshi.</a>
+			</p>
+		</div>
+	</body>	
+</html>	
```

Comment:
remove all extra spaces
Add more content in footer
Add contact us in footer
bad code indentation

Reply:
i will do all the changes